aces/Loris
# [dataquery] Add support for imaging data to new dataquery frontend

This adds support for imaging data to the new data query module. The imaging data doesn't have complete parity with the CouchDB_MRI_Import script, but *does* allow multiple images per session (rather than only the one selected image that was required in the old DQT.)

This partly to get imaging data in the new frontend, and partly a proof-of-concept for adding support for "Cardinality::MANY" data types to the frontend. 

The fields available in this initial pass are:
- ScanDone: Yes/No (per session)
- `$modality` file path (ie. the File value from the files table)
- `$modality` URL (the URL constructed uses the API because that seems to be the most stable place to construct it
- `$modality` QC Status

#### Testing instructions (if applicable)

1. Go to "Data Query Tool (alpha)" under Reports menu
2. Run some queries using imaging fields. No special setup (other than having imaging data) should be required. Data should be accurate and performance should be reasonable.
